build-support/bin/release.sh

```diff
@@ -134,11 +134,18 @@ REQUIREMENTS_3RDPARTY_FILES=(
 # internal backend packages and their dependencies which it doesn't have,
 # and it'll fail. To solve that problem, we load the internal backend package
 # dependencies into the pantsbuild.pants venv.
+#
+# TODO: Starting and stopping pantsd repeatedly here works fine, but because the
+# created venvs are located within the buildroot, pantsd will fingerprint them on
```

Comment:
We do encourage users to create venvs within the build root: https://pants.readme.io/docs/python-third-party-dependencies#tip-set-up-a-virtual-environment-optional

Presumably, those will be in their .gitignore already, so it would be in `pants_ignore`. Should this be okay for end users?

Reply:
This is referring to a virtualenv that contains pants' own code, rather than to end user virtualenvs. pantsd fingerprints its own `sys.path` and source plugins in order to restart if they are edited.